JS/Lesson15/onur/script.js

```diff
@@ -121,10 +57,44 @@ checkIfStringIsUpperCase('write a program that takes in');
 
 // 5. Write a program that takes in a number and checks if it's positive, negative, or zero using if else statement.
 
+function isNumberPositiveOrNegative(num) {
```

Comment:
A note for the future, since JS is not strongly types language, it maybe that the value that is passed to the function is not even a number. In real world, we usually check that the value passed is a number first, if it's not, the function should not even do the checks.